pygtail/core.py

```diff
@@ -59,6 +59,9 @@ def __init__(self, filename, offset_file=None, paranoid=False, copytruncate=Fals
                 # Look for the rotated file and process that if we find it.
                 self._rotated_logfile = self._determine_rotated_logfile()
 
+        # test that this works, allow exceptions to propergate at this point
```

Comment:
Hi,

pygtail module not handle the ams auto rotation log files. Kindly provide the fix for this.

Thanks,
Gopal.